lib/hexpm/block_address/block_address.ex

```diff
@@ -20,8 +20,8 @@ defmodule Hexpm.BlockAddress do
     disallowed =
       Hexpm.BlockAddress.Entry
       |> Hexpm.Repo.all()
-      |> Enum.map(&Hexpm.Utils.parse_ip_mask(&1.ip))
-      |> Enum.reject(fn {ip, _mask} -> ip == nil end)
+      |> Stream.map(&Hexpm.Utils.parse_ip_mask(&1.ip))
+      |> Stream.reject(fn {ip, _mask} -> ip == nil end)
```

Comment:
A for/1 with uniq: true would be even better

Reply:
ok I made those two changes let me know what you think.